gossip/client.go

```diff
@@ -41,7 +41,6 @@ type client struct {
 	rpcClient   *rpc.Client   // RPC client
 	forwardAddr net.Addr      // Set if disconnected with an alternate addr
 	lastFresh   int64         // Last wall time client received fresh info
-	err         error         // Set if client experienced an error
```

Comment:
Why did you remove this? Seems somewhat useful to be able to log this info from where client'a completion is actually handled as opposed to an undifferentiated info log.